Makefile

```diff
@@ -2,26 +2,10 @@ BASE_IMAGE := fedora:latest
 TEST_TARGET ?= ./tests/
 PY_PACKAGE := ogr
 OGR_IMAGE := ogr
-IS_FEDORA := $(shell uname -r | grep -c "\.fc[1-9]")
-ifeq ($(IS_FEDORA), 1)
-    PACKAGE_CHECKER := rpm -q --quiet
-    PACKAGE_INSTALLER := dnf install
-else
-    PACKAGE_CHECKER := :
-    PACKAGE_INSTALLER := :
-endif
-
-prepare-build:
-	if ! $(PACKAGE_CHECKER) ansible-bender ; then sudo $(PACKAGE_INSTALLER) ansible-bender ; fi
-	if ! $(PACKAGE_CHECKER) podman ; then sudo $(PACKAGE_INSTALLER) podman ; fi
 
 build: recipe.yaml
```

Comment:
I think this should be renamed to `build-test-image` in order to be consistent with the rest of the projects. Doing so might also reduce some of the confusion around what the built image is going to be used for (In other projects, `make build` would built a "service" image, which would be used in a deployment. OGR being a library, doesn't have something like this, obviously.)